### DISCUSSION TITLE
Ducktape v0.12.0 Version is now Available

### DISCUSSION THREAD

**Discussion by nagpavan-chilakam** [confluentinc/ducktape#418] [General]
**_We are super happy to announce the release of a new Minor Version of Ducktape - v0.12.0_**

**Whats new in v0.12.0**
**New Support:** 
Added support for newer Python versions 3.10, 3.11 and 3.12 by @ivandasch  in #397. This was requested by different members
**Improved developer experience:**
Store summary of previous runs when deflaking by @imcdo  in #360
Adding nodes used in test summary for HTML report by @abolipai99 in #378
Parse args from config files independently by @imcdo in #395
**Improved stability:**
Stop the hanging threads in runner client forcefully after 30 seconds by @sprsi  in #416
Runner Client Minor Refactor and Test by @imcdo in #366

**ChangeLog:**
Store summary of previous runs when deflaking
Runner Client Minor Refactor and Test
Adding nodes used in test summary for HTML report
Parse args from config files independently
add support to python 3.10, 3.11 and 3.12

**New Contributors**
@abolipai99 in #378
@sprsi  in #416

**Documentation**
https://ducktape.readthedocs.io/en/latest/index.html

**Comment by ivandasch** [comment]
Great! But it seems that it is required to release the new one with critical bug fix

https://github.com/confluentinc/ducktape/issues/419

**Comment by ivandasch** [comment]
The fix is ready -- https://github.com/confluentinc/ducktape/pull/420
